tools/jit/gen_jit_dispatch.py

```diff
@@ -42,6 +42,7 @@
     'Scalar': 'Scalar',
     'MemoryFormat': 'MemoryFormat',
     'MemoryFormat?': 'MemoryFormat?',
+    'QScheme': 'QScheme',
```

Comment:
Umm... why is this in this diff lol

Reply:
You mean QScheme? Probably merge result.